packages/components/filters/filters-list/src/filters-list.stories.tsx

```diff
@@ -1,22 +1,162 @@
-import type { Meta, StoryObj } from '@storybook/react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { MenuProps, MenuListProps } from 'react-select';
+import { useState, useCallback } from 'react';
 import FiltersList from './filters-list';
+import SelectInput from '../../../inputs/select-input';
 
 const meta: Meta<typeof FiltersList> = {
   title: 'components/Filters/FiltersList',
   component: FiltersList,
   tags: ['local-dev'],
-  argTypes: {
-    label: {
-      control: 'text',
-    },
-  },
+  argTypes: {},
 };
 export default meta;
 
-type Story = StoryObj<typeof FiltersList>;
+type Story = StoryFn<typeof FiltersList>;
+
+const CustomSelectMenu = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
+const CustomMenuList = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuListProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
+const primaryColorOptions = [
+  { label: 'Blue', value: 'blue', key: 'blue', id: '2' },
+  { label: 'Red', value: 'red', key: 'red', id: '4' },
+  { label: 'Orange', value: 'orange', key: 'orange', id: '5' },
+  { label: 'Yellow', value: 'yellow', key: 'yellow', id: '6' },
+  { label: 'Green', value: 'green', key: 'green', id: '7' },
+];
+
+const secondaryColorOptions = [
+  { label: 'Purple', value: 'purple', key: 'purple', id: '3' },
+  { label: 'Forest', value: 'forest', key: 'forest', id: '8' },
+  { label: 'Slate', value: 'slate', key: 'slate', id: '9' },
+  { label: 'Silver', value: 'silver', key: 'silver', id: '10' },
+];
+
+export const BasicExample: Story = () => {
+  const [primaryColorValue, setPrimaryColorValue] = useState<string[]>([]);
+  const [secondaryColorValue, setSecondaryColorValue] = useState<string[]>([]);
+
+  const clearPrimaryColorFilter = () => setPrimaryColorValue([]);
+  const clearSecondaryColorFilter = () => setSecondaryColorValue([]);
+
+  const getPrimaryColorTags = useCallback(() => {
+    return primaryColorValue.length > 0
+      ? primaryColorValue.map((value) => ({
+          key: value,
+          label: value.toUpperCase(),
+        }))
+      : [];
+  }, [primaryColorValue]);
+
+  const getSecondaryColorTags = useCallback(() => {
+    return secondaryColorValue.length > 0
+      ? secondaryColorValue.map((value) => ({
+          key: value,
+          label: value.toUpperCase(),
+        }))
+      : [];
+  }, [secondaryColorValue]);
+
+  const clearAllFilters = () => {
+    clearPrimaryColorFilter();
+    clearSecondaryColorFilter();
+  };
+
+  const filters = [
```

Comment:
Where I may need some clarity is if we are only just passing the configs as children or the whole filters array too?

Reply:
I can tell you I spent more time figuring out what should go in as children, thereby creating the subset of data. Syntax-wise, I agree that passing the configuration as props seems more straightforward to follow. It just seems  easier to shape and centralise the data consumed from the stories(users) that way.
I will also document this in the ticket. 
Thank you

Reply:
That's also my takeaway from this, fwiw, thanks @ddouglasz